tools/plugins/com.liferay.ide.server.ui/src/com/liferay/ide/server/ui/action/AbstractServerRunningAction.java

```diff
@@ -59,6 +75,32 @@ protected Shell getActiveShell()
 
     protected abstract int getRequiredServerState();
 
+	public void run(IAction action) {
+
+		if (selectedServer != null) {
+			IRuntime runtime = selectedServer.getRuntime();
+
+			IStatus validate = runtime.validate(new NullProgressMonitor());
+
+			if (!validate.isOK()) {
+				MessageDialog dialog = new MessageDialog(
+					getActiveShell(), "Invalid server", null,validate.getMessage(), 
```

Comment:
This text should be "Server runtime configuration invalid"

Then button options

[Edit runtime configuration, Cancel]